src/Plugin/EntityReferenceSelection/OgSelection.php

```
@@ -60,7 +64,14 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
     // the default selection handler of the entity, which the field reference
     // to, and add another logic to the query object i.e. check if the entities
     // bundle defined as group.
-    $query = $this->getSelectionHandler()->buildEntityQuery();
+    $selection_handler = $this->getSelectionHandler();
+
+    if (empty($selection_handler)) {
+      return $query = parent::buildEntityQuery($match, $match_operator);
```

Maybe throw an exception instead? Otherwise we silently fail here.